### PR TITLE
DigitalOcean dynamic DNS

### PR DESCRIPTION
**Pull request by dheishman**
I added basic DigitalOcean dynamic DNS support to the dyndns plugin. It requires knowing the record ID of the domain, which unfortunately means you won't see the actual name of the record being updated. It works well enough for my purposes, though, and you can always put the record name in the description field for reference.

### CODE REVIEW

File: dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc [hunk 1]
```diff
@@ -39,6 +39,7 @@
  *    - Oray (oray.com)
  *    - regfish (regfish.de)
  *    - regfish IPv6 (regfish.de)
+ *    - DigitalOcean (digitalocean.com)
```
Comment (fichtner) [inline dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc]: there is a "last tested" line in this file as well, would you mind filling it out?

**Comment by fichtner** [comment]
@dheishman thanks, nice work! minor request with the last tested annotation otherwise it can go in :)

**Comment by dheishman** [comment]
Oops, missed that line! It's added now

**Comment by fichtner** [comment]
Merged, thanks a lot ❤️

**Pull request merged**